crates/matrix-sdk/src/sliding_sync/mod.rs

```diff
@@ -292,7 +292,8 @@ impl SlidingSync {
             response_processor.process_and_take_response().await?
         };
 
-        debug!(?sync_response, "Sliding Sync response has been handled by the client");
+        debug!("Sliding Sync response has been handled by the client");
+        trace!(?sync_response, "Sliding sync response");
```

Comment:
```suggestion
        trace!(?sync_response);
```

You _could_ also create an event without a message.

Reply:
Thanks, yeah I wasn't 100% sure whether it was allowed and thus tried to write something meaningful but didn't really manage it :-)